### PR TITLE
Fixes the eminence getting spammed by blessed tiles.

### LINKED ISSUE #34840: ClockCult eminence gets spammed by holy water tiles
Needs a cooldown

### PR DESCRIPTION
**Pull request by DaxDupont**
[Changelogs]: 

:cl: Dax Dupont
fix: Eminence won't get spammed by tile crossing now.
/:cl:

[why]: fixes https://github.com/tgstation/tgstation/issues/34840

### CODE REVIEW

File: code/game/gamemodes/clock_cult/clock_mobs/_eminence.dm [hunk 1]
```diff
@@ -26,15 +27,19 @@
...
-			to_chat(src, "<span class='warning'>This turf is consecrated and can't be crossed!</span>")
+			if ((REALTIMEOFDAY - lastWarning)) >= 30) 
+				lastWarning = REALTIMEOFDAY
+				to_chat(src, "<span class='warning'>This turf is consecrated and can't be crossed!</span>")
 			return
 		if(!GLOB.ratvar_awakens && istype(get_area(T), /area/chapel))
-			to_chat(src, "<span class='warning'>The Chapel is hallowed ground under a heretical deity, and can't be accessed!</span>")
+			if ((REALTIMEOFDAY - lastWarning)) >= 30) 
+				lastWarning = REALTIMEOFDAY
+				to_chat(src, "<span class='warning'>The Chapel is hallowed ground under a heretical deity, and can't be accessed!</span>")
```
Comment (coiax) [inline code/game/gamemodes/clock_cult/clock_mobs/_eminence.dm]: @Cyberboss this is such a common pattern, I feel like there should be a component for it.

@DaxDupont that's not a criticism of this change, it's fine

**Comment by DaxDupont** [comment]
wait merge conflict fuuuuuuuuuuuuuuuuuuuu

**Comment by optimumtact** [comment]
why not just use world.time

**Comment by DaxDupont** [comment]
I guess I could, just used that before

**Pull request merged**